documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,53 +1,21 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} automatically creates a Prometheus `ServiceMonitor` that scrapes metrics from your {brandname} cluster.
 
-.Prerequisites
+.Procedure
 
-* Have an `oc` client.
 * Enable monitoring for user-defined projects on {ocp}.
 
-.Procedure
+When it detects that the Prometheus Operator is installed, {ispn_operator} adds the `infinispan.org/monitoring: 'true'` annotation to your `Infinispan` CR.
```

Comment:
Can we change Prometheus Operator to `ServiceMonitor`? Eg: `When ServiceMonitor resource presence is detected on the cluster, {ispn_operator}...`. With user workload monitoring you don't exactly deal with the Prometheus Operator and it may be confusing

Reply:
Good point, Pavel. This might also be confusing with the note that we make about the community Prometheus Operator. I've changed the wording.